text/0012-cva.md

```diff
@@ -0,0 +1,161 @@
+
+## Reference-level explanation
+
+[reference-level-explanation]: #reference-level-explanation
+
+Establishment of a Casper Validator's Association plus associated legal, technical and communications infrastructure is considered primarily a soft design problem.  It assumes that the proposal would be accepted by prospective Genesis set validators - by no means a given.  The design solution necessarily combines a diverse skillset: 
+
+- legal expertise;
+- community building;
+- smart contract development (see VDAO);
+- long-term thinking;
+
+Successful implementation will require clear signals from the CasperLabs management team being transmitted to the core set of validators assisting in the incubation of the network.  If stakeholders can relatively quickly arrive at consensus on both the broad and finer strokes of a proposal, then the association  may be established in a timely manner, i.e. before main-net.  It may even be possible to establish an embryonic VDAO before main-net but more likely this would be released at a later date. 
+ 
+## Drawbacks
+
+[drawbacks]: #drawbacks
+
+1. The proposals outlined in this CEP are potentially problematic from a political & jurisdictional perspective.  The crypto space is unique in that attracts a fair number of non-regulated entities, such entities may very likely be put off by the idea of mandatory association membership.
+
+2. The notion that validators may be classified in some respect, e.g. regulated | certified | other, requires some form of arbiter.  This is problematic if, for example, the arbiter is found to be byzantine.  
+
+## Rationale and alternatives
+
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+In respect of establishing a validator association this is considered to be a partial formalisation of the hitherto proposed 'social consensus'.  It is the assertion of the CEP author that not visibly addressing AML/CFT concerns will result in a reduction in both the amount of capital and prestige of entities committed to the network.  
+
+The impact of not forming an association is that political cliques will inevitably form.  Such cliques focus upon short-medium term tactical objectives to the detriment of coherent long-term strategic objectives.  Furthermore those isolated from cliques, possibly due to simple issues regarding language & culture, may feel somewhat alienated and hence less committed to participating in the network over the long term. 
+
+## Prior art
+
+[prior-art]: #prior-art
+
+- Due to it's potential reach, the Libra project has been forced to think deeply upon the regulatory setting within which the network operators conduct business.  It has formed [an association](https://libra.org/en-US/white-paper/#the-libra-association) in order to navigate a complex regulatory landscape.
```

Comment:
Libra is a much bigger target than any other cryptocurrency because of Zuckerberg's awkward political situation.